ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.hpp

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include <array>
```

Comment:
What happened here?

Reply:
Compile error when PCH are disabled.  Must've been introduced after I did the original sweep in #13700 .